Makefile

```diff
@@ -306,3 +311,10 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+
+GINKGO = $(shell pwd)/bin/ginkgo
+.PHONY: ginkgo
+ginkgo: ## Download ginkgo locally if necessary.
+ifeq (, $(wildcard $(GINKGO)))
+	$(call go-install-tool,$(GINKGO),github.com/onsi/ginkgo/v2/ginkgo@v2.1.3)
```

Comment:
Please update also go.mod file to include ginkgo version 2

Reply:
As I mentioned, the go.mod is part of the PR #28

Reply:
I think it's better to include it as part of that PR, I have a  PR with UT that includes changes in the Makefile, vendor, and go.mod file

Reply:
V2 is included from #28 and removed here. @bardielle can we consider this conversion resolved?